css-view-transitions-1/Overview.bs

```diff
@@ -1049,17 +1049,17 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			which causes the transition to [=skip the view transition|skip=].
 			[Discussion of this behavior](https://github.com/w3c/csswg-drafts/issues/8045).
 
-		: <dfn>old state captured steps</dfn>
+		: <dfn>process old state captured</dfn>
```

Comment:
We need to update the call-sites which invoke this algorithm too.

Reply:
They're all updated in this PR.